### PR TITLE
docs: add information about PYTHONPATH

### PR DESCRIPTION
**Pull request by zshipko**

### CODE REVIEW

**Review by G4Vi — APPROVED** [review]
LGTM, good idea to document this!

File: README.md [hunk 1]
```diff
@@ -60,6 +60,37 @@ The goal of writing an
...
+```bash
+$ python3 -m virtualenv ./deps
+$ source ./deps/bin/activate
+$ pip install toml
+```
+
+And then point `extism-py` to the dependencies
+
+```bash
+$ PYTHONPATH=./deps/lib/python3.12/site-packages extism-py -o a.wasm plugin.py
```
Comment (nilslice) [inline README.md:88]: is there a generally easy way to get `./deps/lib/python3.12/site-packages` from some CLI command we can recommend the user calls here, or is this just "`cd` around the locals `./deps` directory for where your `site-packages` is?

Comment (zshipko) [reply README.md:95]: Good call, I think this works:

```
 PYTHONPATH=$(python3 -c "import sys; print(sys.path[-1])") extism-py -o a.wasm plugin.py
```

Comment (nilslice) [reply README.md:95]: nice! thank you

**Review by nilslice — APPROVED** [review]

**Pull request merged**